apps/editing-toolkit/editing-toolkit-plugin/premium-content/blocks/buttons/edit.js

```diff
@@ -91,14 +82,13 @@ function ButtonsEdit( { context, subscribeButton, setSubscribeButtonPlan } ) {
 	return (
 		// eslint-disable-next-line wpcalypso/jsx-classname-namespace
 		<Block.div className="wp-block-buttons">
-			<AlignmentHookSettingsProvider value={ alignmentHooksSetting }>
-				<InnerBlocks
-					allowedBlocks={ ALLOWED_BLOCKS }
-					template={ isPreview ? previewTemplate : template }
-					__experimentalMoverDirection="horizontal"
-					templateInsertUpdatesSelection={ false }
-				/>
-			</AlignmentHookSettingsProvider>
+			<InnerBlocks
+				allowedBlocks={ ALLOWED_BLOCKS }
+				template={ isPreview ? previewTemplate : template }
+				templateInsertUpdatesSelection={ false }
+				__experimentalLayout={ { type: 'default', alignments: [] } }
```

Comment:
I'm not sure this line actually does anything (see PR description, section 'Open questions'). We could consider removing it.